packages/lesmis/src/api/queries/useOrgUnitData.js

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This is a place holder query until EntityServer is ready and so is outside the scope of this review.